hack/tools/install.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
+
+: ${KUBE_NAMESPACE:=m4d-system}
+
+source secret-provider/deploy/vault-util.sh
+
+kubectl create ns $KUBE_NAMESPACE || true
+
+kubectl config set-context --current --namespace=$KUBE_NAMESPACE
```

Comment:
What relies on m4d-system to be the current namespace?

Reply:
You could in theory install to a different ns if you want. The only thing the currently "hard-coded" (but I guess could be modified using helm) is as far as I know in the vault YAML:

Reply:
I didn't mean to ask if we can install to a different namespace (we can't yet). I asked why does it need to be the current namespace. Asking because @hunchback took the approach of always specifying the target namespace (`--namespace`/`-n` and the KUBE_NAMESPACE variable)

Reply:
It seems the installation of some of the components assume that we've set `m4d-system` to be the current ns. As we discussed, I'll remove that line from the script and put it in the README.